src/assets/stylesheets/main/_api.scss

```diff
@@ -45,7 +45,7 @@ $objinfo-icon-size: 16px;
         font-weight: 700;
       }
 
-      a.reference > .sig-name {
+      a.reference > span {
```

Comment:
I guess this is okay but can you explain more why this is needed?

Reply:
Its related to the differences in domain. I found that reference links in C++ domain did not get special coloring (the intention of this CSS rule) when the `highlight` class (which is needed for other syntax highlighting like `class`) is added to the signature.